### DISCUSSION TITLE
How to set up a domain name to automatically redirect from HTTP to HTTPS?

### DISCUSSION THREAD

**Discussion by cui8** [NginxProxyManager/nginx-proxy-manager#3294] [Q&A]
For example: Visiting the website http://ccc.top automatically redirects to https://ccc.top, without the user having to manually enter https. This has been troubling me for days. Can everyone please help me out?

**Comment by republicandaddy** [comment]
For me, I just used the SSL tab of the proxy host entry screen.  There you'll see a switch for "Force SSL", but this assumes you're using an SSL cert.....

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/71984381/6ba1e6b1-7bdf-471c-b428-0eb4f844ee52)
